src/models/appointments.js

```diff
@@ -0,0 +1,28 @@
+const { Schema, model } = require('mongoose');
+
+const appointmentSchema = Schema(
+	{
+		date: {
+			type: Date,
+		},
+		hour: {
+			type: String,
+		},
+		name: {
+			type: String,
+		},
+		phone: {
+			type: String,
+		},
+		email: {
+			type: String,
+		},
+	},
+	{ versionKey: false, timestamps: true },
+);
+
+const Appointment = model('appointment', appointmentSchema);
+
+module.exports = {
+	Appointment,

```

Comment:
Hey!
Why do u export the whole object, if you're needed to export just one var?

Reply:
need to discuss

Reply:
e.g. ```module.exports = Appointment;```
It's all.

Reply:
done